src/com/sun/ts/tests/signaturetest/SignatureTestDriver.java

```diff
@@ -98,26 +79,7 @@ public String getPackageFileImpl(String binDir) {
    */
   public String getMapFileImpl(String binDir) {
 
-    String theMapFile;
-
-    if (isJavaSEVersion("1.5")) {
-      // we have JavaSE5 so use that map file
-      theMapFile = "sig-test_se5.map";
-    } else if (isJavaSEVersion("1.6")) {
-      theMapFile = "sig-test_se6.map";
-    } else if (isJavaSEVersion("1.7")) {
-      theMapFile = "sig-test_se7.map";
-    } else if (isJavaSEVersion("1.8")) {
-      theMapFile = "sig-test_se8.map";
-    } else if (isJavaSEVersion("9")) {
-      theMapFile = "sig-test_se9.map";
-    } else if (isJavaSEVersion("11")) {
-      theMapFile = "sig-test_se11.map";
-    } else {
-      // we didn't properly identify the java se version being used so
-      // we will try to use the sig-test.map file.
-      theMapFile = "sig-test.map";
-    }
+    String  theMapFile = "sig-test_se8.map";
```

Comment:
Nit : If this will be the only mapping file available , could we remove the _se8 from the map file, same for sig-test-pkg-list_se8.txt.

Reply:
I think that we could do that so we don't have to rename the file again in the future.